include/wx/stc/completionitem.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _WX_STC_COMPLETIONITEM_H_
+#define _WX_STC_COMPLETIONITEM_H_
+
+#include "wx/string.h"
+#include "wx/clntdata.h"
+
+class WXDLLIMPEXP_STC wxCompletionItem
+{
+public:
+    wxCompletionItem(const wxString &name, int icon, const wxString &doc = wxString(), wxClientData *data = NULL);
+    virtual ~wxCompletionItem();
+
+    const wxString &GetName();
+    int GetIcon();
+    const wxString &GetDocumentation();
+    wxClientData *GetClientObject();
+
+protected:
```

Comment:
Please use `private` unless you really want to allow derived class changing the members directly (and, if so, why?).

Reply:
good catch :)